scripts/integtest.sh

```diff
@@ -95,18 +95,19 @@ fi
 USERNAME=`echo $CREDENTIAL | awk -F ':' '{print $1}'`
 PASSWORD=`echo $CREDENTIAL | awk -F ':' '{print $2}'`
 
-OPENSEARCH_HOME=`ps -ef | grep -o "[o]pensearch.path.home=\S\+" | cut -d= -f2- | head -n1`
+OS="`uname`"
 
-curl -SL https://raw.githubusercontent.com/opensearch-project/sql/main/integ-test/src/test/resources/datasource/datasources.json -o "$OPENSEARCH_HOME"/datasources.json
-
-yes | $OPENSEARCH_HOME/bin/opensearch-keystore add-file plugins.query.federation.datasources.config $OPENSEARCH_HOME/datasources.json
-
-if [ $SECURITY_ENABLED == "true" ] 
+if [ $OS != "WindowsNT" ]
```

Comment:
windows doesn't have a uname or bash, I remember different uname gives different outputs on windows. which uname is this? maybe document prerequisite dependencies in case user wants to run tests on windows

Reply:
This requires Cygwin or MinGW packages and these are installed in the ami used in our build systems. Will mention it as a comment.